gprofiler/merge.py

```diff
@@ -31,6 +33,18 @@
 FRAME_REGEX = re.compile(r"^\s*[0-9a-f]+ (.*?) \((.*)\)$")
 
 
+@dataclass
+class EnrichmentOptions:
+    """
+    Profile enrichment options.
+    """
+
+    container_names: bool  # Include container names for each stack in result profile
+    container_names_in_protocol: bool  # Are container names a part of the protocol? (true iff not v1)
```

Comment:
I was expecting this comment :sad:
As mentioned in https://github.com/Granulate/gprofiler/pull/299#issuecomment-1063493241, I actually tried to break the current scheme of `container_names` which complicates this. But I decided not to, too many changes in one PR.

Anyway, this is indeed not an "enrichment option", but it does relate. Now I've placed the `profile_api_version` here.
I don't think it implies a bad design, since those options indeed work together. it's just bad naming. `EnrichmentContext`?

Reply:
Decided f2f to keep it